designer/server/src/routes/forms/create.js

```diff
@@ -183,7 +192,9 @@ export default [
     method: 'POST',
     path: '/create/team',
     async handler(request, h) {
-      const { payload, yar } = request
+      const { auth, payload, yar } = request
+      const { credentials } = auth
+      const { user } = credentials
```

Comment:
Just a reminder that this isn't correctly typed by Hapi

Sometimes `credentials` can be null so we'll need to check before destructuring

```suggestion
      const { user } = credentials
```